exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/DatabaseMigrationTests.kt

```diff
@@ -257,11 +261,585 @@ class DatabaseMigrationTests : DatabaseTestsBase() {
                 SchemaUtils.create(testTableWithIndex)
                 assertTrue(testTableWithIndex.exists())
 
-                val statements = MigrationUtils.statementsRequiredForDatabaseMigration(testTableWithoutIndex)
+                val statements = MigrationUtils.statementsRequiredForDatabaseMigration(testTableWithoutIndex, withLogs = false)
                 assertEquals(1, statements.size)
             } finally {
                 SchemaUtils.drop(testTableWithIndex)
             }
         }
     }
+
+    @Test
+    fun testAddAutoIncrementToExistingColumn() {
+        val tableWithoutAutoIncrement = object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").entityId()
+        }
+        val tableWithAutoIncrement = object : IdTable<Long>("test_table") {
+            override val id: Column<EntityID<Long>> = long("id").autoIncrement().entityId()
+        }
```

Comment:
Could the majority of these identical test tables be extracted to common class objects and shared between the new tests?